build.gradle

```diff
@@ -77,7 +75,7 @@ ext {
     spring4Version = '4.3.7.RELEASE'
     spring5Version = '5.1.9.RELEASE'
     springboot1Version = '1.5.2.RELEASE'
-    springboot2Version = '2.1.8.RELEASE'
```

Comment:
Couldn't upgrade gradle without upgrading springboot due to the SB gradle plugin violating newer gradle rules. Upgrading SB then caused various other things to fail, as SB made some breaking changes for us. That's why this PR has some SB-related stuff in it.